ArduPlane/mode_autoland.cpp

```diff
@@ -138,12 +148,18 @@ bool ModeAutoLand::_enter()
     cmd_loiter.content.location = land_start;
     cmd_loiter.content.location.offset_bearing(plane.takeoff_state.initial_direction.heading + bearing_offset_deg, corrected_loiter_radius);
     cmd_loiter.content.location.loiter_ccw = bearing_err_deg>0? 1 :0;
-
+#if AP_TERRAIN_AVAILABLE
+    if (plane.terrain_enabled_in_current_mode()) {
+        cmd_loiter.content.location.terrain_alt = 1; // this allows terrain following to it as a glide slop destination
```

Comment:
i'm surprised you didn't need to subtract home alt here, I will need to look at this under gdb